examples/todolist/src/Server.hs

```diff
@@ -39,49 +40,53 @@ reset i t = alwaysOk $ do
   atomically $ do
     writeTVar i 0
     writeTVar t []
-  pure $ MessageId 0 -- returns nothing
+  pure $ MessageId Nothing -- returns nothing
```

Comment:
Perfect, now all makes sense! Maybe we can remove the comment now 😉

Reply:
Yay!